releases/rust/db_esdk/Cargo.toml

```
@@ -24,7 +24,7 @@ aws-sdk-kms = "1.50.0"
 aws-smithy-runtime-api = {version = "1.7.3", features = ["client"] }
 aws-smithy-types = "1.2.9"
 chrono = "0.4.38"
-dafny_runtime = { path = "./dafny_runtime_rust"}
+dafny-runtime = "0.1.0"
```

Is it v0.1.0 or v0.1.1?
```suggestion
dafny-runtime = "0.1.1"
```

---

It doesn't matter, 0.1.1 was just for a typo in the README.
Specifying 0.1.0 will still get you 0.1.1 under most circumstances.